Formula/csvtk.rb

```diff
@@ -0,0 +1,17 @@
+class Csvtk < Formula
+  desc "A cross-platform, efficient and practical CSV/TSV toolkit in Golang"
```

Comment:
```suggestion
  desc "Cross-platform, efficient and practical CSV/TSV toolkit in Golang"
```
Descriptions shouldn't begin with articles, that's why the audit fails.